.env.development

```
@@ -9,8 +9,13 @@ JWT_SECRET="my-secret-dev"
 JWT_ACCESS_TOKEN_EXPIRATION_TIME_IN_MINUTES=60
 JWT_REFRESH_TOKEN_EXPIRATION_TIME_IN_DAYS=30
 
+# Vendor API keys
+TADDY_USER_ID=""
```

dotenv-linter (3.3.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)